core/views.py

```diff
@@ -1,7 +1,65 @@
 from knox.views import LoginView as KnoxLoginView
 from rest_framework.authentication import BasicAuthentication
+from core.models import Tag, Source
+from core.serializers import TagSerializer, UserSerializer, SourceSerializer
+from rest_framework import generics, permissions
+from django.contrib.auth.models import User
+from core.permissions import IsOwnerOrReadOnly
+
+
+class UserList(generics.ListAPIView):
+    authentication_classes = [BasicAuthentication]
```

Comment:
Change all these to use `TokenAuthentication`... can't be entering user credentials for every request.

`from knox.auth import TokenAuthentication`